presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -70,6 +70,15 @@ When enabled, the logical plan will begin to be built and validated before
 queueing and allocation of cluster resources so that any errors or
 incompatibilities in the query plan will fail quickly and inform the user.
 
+``single-node-execution-enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+This property enables running the query on a single node under a distributed cluster setup.
```

Comment:
```suggestion
This property ensures that queries scheduled in this cluster use only a single node for execution, which may improve performance for small queries which can be executed within a single node.
```

The above is just a suggestion.  Avoid referring to "the query", because a config property is global and affects "all queries".  I also wouldn't presume the reader has knowledge on what is considered to be a low latency use case, so I'd drop the definite article, and I'd try to qualify what counts as a small/tiny query in some way.